backend/settings.py

```diff
@@ -157,16 +157,16 @@
     }
 }
 
-# This is needed to allow the LTI tool to work with the Django cache when running on multiple workers
 CACHES = {
     "default": {
-        "BACKEND": "django_mysql.cache.MySQLCache",
-        "LOCATION": "django_clrt_cache",
-        "KEY_PREFIX": "clrt",
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": config('REDIS_LOCATION', "redis://127.0.0.1:6379/1"),
```

Comment:
Can we have the value `redis://clrt_redis_host:6379/1` instead of `redis://127.0.0.1:6379/1` as default?

 Since without the `REDIS_LOCATION` in the `.env` the 127... url throw `redis.exceptions.ConnectionError: Error 111 connecting to 127.0.0.1:6379. Connection refused.`

Reply:
Yeah sure we can make this the default here 
```suggestion
        "LOCATION": config('REDIS_LOCATION', "redis://clrt_redis_host:6379"),
```

Actually, I think we might be even able to drop the dependency and use 
`django.core.cache.backends.redis.RedisCache` which should be built-in now as of Django 4.0. Let me make that change too.